cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```
@@ -53,13 +53,15 @@ data:
         text/x-cross-domain-policy;
 
     upstream model {
-{{- if .Values.kubecostFrontend.useDefaultFqdn }}
-        server {{ $serviceName }}.{{ .Release.Namespace }}.svc.cluster.local:9003;
-{{- else if (.Values.kubecostFrontend.model).fqdn }}
-        server {{ .Values.kubecostFrontend.model.fqdn }};
-{{- else }}
-        server {{ $serviceName }}.{{ .Release.Namespace }}:9003;
-{{- end }}
+        {{- if .Values.kubecostFrontend.useDefaultFqdn }}
+        server {{ .Release.Name }}-aggregator.{{ .Release.Namespace }}.svc.cluster.local:9004;
+        {{- else }}
+        {{- if (.Values.kubecostFrontend.aggregator).fqdn }}
+        server {{ .Values.kubecostFrontend.aggregator.fqdn }};
+        {{- else }}
+        server {{ .Release.Name }}-aggregator.{{ .Release.Namespace }}:9004;
+        {{- end }}
+        {{- end }}
```

Minor, but I think more readable?

```suggestion
        {{- if .Values.kubecostFrontend.useDefaultFqdn }}
        server {{ .Release.Name }}-aggregator.{{ .Release.Namespace }}.svc.cluster.local:9004;
        {{- else if (.Values.kubecostFrontend.aggregator).fqdn }}
        server {{ .Values.kubecostFrontend.aggregator.fqdn }};
        {{- else }}
        server {{ .Release.Name }}-aggregator.{{ .Release.Namespace }}:9004;
        {{- end }}
```